fighters/bayonetta/src/opff.rs

```diff
@@ -266,32 +245,46 @@ unsafe fn recovery_resource_management(fighter: &mut L2CFighterCommon) {
     
 }
 
+unsafe fn clear_proration(fighter: &mut L2CFighterCommon) {
```

Comment:
This won't catch all landing situations, if that's what you're trying to do.
Better off doing `if prev_situation_kind == air && situation_kind == ground && [damage statuses]`